src/StatusQ/Core/StatusScrollView.qml

```diff
@@ -63,4 +63,15 @@ Flickable {
         policy: ScrollBar.AsNeeded
         visible: resolveVisibility(policy, root.height, root.contentHeight)
     }
+
+    function ensureVisible(rect) {
```

Comment:
We have an identical method in `StatusBaseInput.qml` and sth similar in `StatusSelectableText.qml`. I would suggest to move this method to utils.

Reply:
@micieslak Yes - I observed that and I believe it is identical. The thing is those functions are also inside of Flickable same as here. So I guess we dont need to move it to utils, but maybe StatusScrollView must be reused in place of those Flickable and have function inside of shared component.

Reply:
I would opt for standalone function in utils taking Flickable as a first argument. It gives slightly more flexibility. On top of that we could add ensureVisible to StatusScrollView as well and use standalone version for implementation:
```
function ensureVisible(rect) {
   StatusUtils.ensureVisible(this, rect)
}
```
Than we are safe in case StatusScrollView is refactored and decided to rely on different top level component than Flickable.

Reply:
Ok - agree